src/redisraft.c

```diff
@@ -742,7 +743,9 @@ static void handleRedisCommandAppend(RedisRaftCtx *rr,
      * we're still a leader. Otherwise, just process the reads. */
     if (cmd_flags & CMD_SPEC_READONLY && !(cmd_flags & CMD_SPEC_WRITE)) {
         if (!rr->config.quorum_reads) {
-            RaftExecuteCommandArray(rr, ctx, ctx, cmds);
+            RaftReq req = {0};
+            req.ctx = ctx;
```

Comment:
```suggestion
            RaftReq req = {
                .ctx = ctx
            };
```
I don't know which one you like more but you can use this version if you want, just fyi.